goalie/error_estimation.py

```diff
@@ -59,7 +58,15 @@ def form2indicator(F: ufl.form.Form) -> Function:
         dx = firedrake.dx(integral.subdomain_id())
         cell_terms += p0test * integral.integrand() * dx
     if cell_terms != 0:
-        indicator += firedrake.assemble(cell_terms)
+        cell_contrib = Function(P0)
+        sp = {
+            "snes_type": "ksponly",
+            "ksp_type": "preonly",
+            "pc_type": "lu",
+            "pc_factor_mat_solver_type": "mumps",
+        }
+        firedrake.solve(mass_term == cell_terms, cell_contrib, solver_parameters=sp)
+        indicator += cell_contrib
```

Comment:
Question that came up in the meeting: what's the difference between this and the original, other than the type? Is this system also trivial? If so, we could solve it using a simple application of a Jacobi preconditioner, as above.

Reply:
Sorry, probably misunderstood yesterday: this is indeed changed now: before you add in a cell-integration of the integrand, but now you're adding in a cell-average (i.e. divided by cell area - or equivalently this is just a projection into P0). Don't know which one is correct

Reply:
Ah yes, good point. Considering this a bit more, I think this is the right thing to do in f55dce1 - multiply throughout the RHS with `CellSize(mesh)`.